correlations-matrix/readme.md

```diff
@@ -0,0 +1,18 @@
+# Correlation matrix CSV
```

Comment:
Strictly, we are not generating a CSV anywhere.
In fact, how could a user automatically create a csv from correlations-tab output?

Reply:
Yes, that's a matrix currently. I think it might be more useful if we finally output a CSV, so I'll try to change that.

Reply:
Now that I changed it, I remember why I used the matrix finally. When we create a text as output, I don't see how to download the entire contents of it. I only see it truncated. @osroca ?